src/core/fonts.js

```diff
@@ -433,11 +433,14 @@ function convertCidString(charCode, cid, shouldThrow = false) {
  * font that we build
  * 'charCodeToGlyphId' - maps the new font char codes to glyph ids
  */
-function adjustMapping(charCodeToGlyphId, hasGlyph, newGlyphZeroId) {
+function adjustMapping(charCodeToGlyphId, hasGlyph, newGlyphZeroId, toUnicode) {
   const newMap = Object.create(null);
+  const toUnicodeExtraMap = new Map();
   const toFontChar = [];
+  const usedGlyphIds = new Set();
   let privateUseAreaIndex = 0;
-  let nextAvailableFontCharCode = PRIVATE_USE_AREAS[privateUseAreaIndex][0];
+  const privateAreaStart = PRIVATE_USE_AREAS[privateUseAreaIndex][0];
```

Comment:
For consistency with the name used just below, this should probably be:
```suggestion
  const privateUseOffetStart = PRIVATE_USE_AREAS[privateUseAreaIndex][0];
```